apps/quickstart-integrations-monday-code/index.js

```diff
@@ -87,6 +88,35 @@ app.post(
   }
 );
 
+
+app.post(
+    "/produce",
+    async (req, res) => {
+        try {
+            const { body } = req;
+            const messageId = await produceMessage({ body })
+            return res.status(200).send({ messageId });
+        } catch (err) {
+            logger.error(JSON.stringify(err));
+            return res.status(500).send({ message: "internal server error" });
+        }
+    }
+);
+
+app.post(
+    "/mndy-queue",
+    async (req, res) => {
+        try {
+            const { headers, body, query } = req;
+            readQueueMessage({ headers, body, query });
+            return res.status(200).send(); // return 200 to ACK the queue message
```

Comment:
let's return the same message as was sent:
```suggestion
            return res.status(200).send(body); // return 200 to ACK the queue message
```

Reply:
why?, we just need to return 200 in order ACK the message so it will be deleted from the queue?, no reason to send back the body